packages/freetype.rb

```diff
@@ -78,5 +136,63 @@ def self.postinstall
         end
       end
     end
+      return if File.file?("#{CREW_LIB_PREFIX}/#{@libname}.la")
+
+    # Create libtool file. Needed by
+    @libname = name.to_s.start_with?('lib') ? name.downcase : "lib#{name.downcase}"
+    @libnames = Dir["#{CREW_LIB_PREFIX}/#{@libname}.so*"]
+    @libnames = Dir["#{CREW_LIB_PREFIX}/#{@libname}-*.so*"] if @libnames.empty?
+    @libnames.each do |s|
+      s.gsub!("#{CREW_LIB_PREFIX}/", '')
+    end
+    @dlname = @libnames.grep(/.so./).first
+    @libname = @dlname.gsub(/.so.\d+/, '')
+    @longest_libname = @libnames.max_by(&:length)
+    @libvars = @longest_libname.rpartition('.so.')[2].split('.')
+
+    @libtool_file = <<~LIBTOOLEOF
+      # #{@libname}.la - a libtool library file
+      # Generated by libtool (GNU libtool) (hacked up by Chromebrew)
+      #
+      # Please DO NOT delete this file!
+      # It is necessary for linking the library.
+
+      # The name that we can dlopen(3).
+      dlname='#{@dlname}'
+
+      # Names of this library.
+      library_names='#{@libnames.reverse.join(' ')}'
+
+      # The name of the static archive.
+      old_library='#{@libname}.a'
+
+      # Linker flags that cannot go in dependency_libs.
+      inherited_linker_flags=''
+
+      # Libraries that this one depends upon.
+      dependency_libs=''
+
+      # Names of additional weak libraries provided by this library
+      weak_library_names=''
+
+      # Version information for #{name}.
+      current=#{@libvars[1]}
+      age=#{@libvars[1]}
+      revision=#{@libvars[2]}
+
+      # Is this an already installed library?
+      installed=yes
+
+      # Should we warn about portability when linking against -modules?
+      shouldnotlink=no
+
+      # Files to dlopen/dlpreopen
+      dlopen=''
+      dlpreopen=''
+
+      # Directory that this library needs to be installed in:
+      libdir='#{CREW_LIB_PREFIX}'
+    LIBTOOLEOF
+    File.write("#{CREW_LIB_PREFIX}/#{@libname}.la", @libtool_file)
```

Comment:
Why is this process in postinstall instead of install?  These files will not be included in the package.

Reply:
I'll rebuild these...